static/aurora/v1-chinook-deployment-template.json

```diff
@@ -687,6 +687,108 @@
         },
         "description": null
       },
+      "delete_playlist_track": {
```

Comment:
I got errors complaining these native queries were missing from the deployment while I was developing, and so I put that down to a botched merge while I was preparing version 2 of the deployments.

But in this diff I appear to be the one introducing them. Any clarifying info is very welcome.

Reply:
I have not yet tested native query mutations for other databases. I was planning on adding more tests but it got delayed because of other stuff.

Reply:
Ah ok. I'm fine to leave these in if you like.

Reply:
I'm fine with adding or not adding them in this commit. Whatever is less work for you.